Resources/ServerInfo/_NF/Guidebook/Shipyard/Kestrel.xml

```diff
@@ -81,12 +84,20 @@
   
   - Check that the gyroscope is anchored, powered, and enabled.
   - Check that the mini gravity generator is anchored, powered, and enabled.
-  - Check that the bow blast doors are closed.
+  - Check that the blast doors are closed.
   
   ## Sidenotes
   
-  * - Kestrel-class salvage ships are equipped with five APC units that can be used to appraise the ship's total power consumption. Unmodifued Kestrel-class ship requires 27.6 kW of power to remain operational.
+  * - Kestrel-class salvage ships are equipped with five APC units that can be used to appraise the ship's total power consumption. Unmodifued Kestrel-class ship requires 28.6 kW of power to remain operational.
   
-  ** - Kestrel-class salvage ships have higher power demand than single standard P.A.C.M.A.N. generator unit can provide at optimal settings, however the ship comes equipped with two P.A.C.M.A.N. generator units, recommended target power output for each P.A.C.M.A.N. generator unit is 15 kW.
+  ** - Kestrel-class salvage ships have higher power demand than standard P.A.C.M.A.N. generator unit can provide at optimal settings, so the standard P.A.C.M.A.N. was replaced with S.U.P.E.R.P.A.C.M.A.N.: recommended Target Power value to avoid blackouts during Gravity Generator charging sequence is 30 kW.
```

Comment:
Consistency with the Bulker.
```suggestion
  ** - Kestrel-class salvage ships have higher power demand than a standard P.A.C.M.A.N. can provide at optimal settings, and come equipped with a S.U.P.E.R.P.A.C.M.A.N. instead. Please note that the recommended Target Power value to avoid blackouts during Gravity Generator charging sequence is 30 kW.
```